angular/projects/common/src/lib/common.component.ts

```diff
@@ -2,11 +2,6 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'lib-common',
-  template: `
-    <em>
-      common component!
-    </em>
-  `
+  template: ` <em> common component! </em> `,
```

Comment:
Can we avoid this one somehow?

Reply:
It seems it is not supported by `prettier` for now. https://github.com/prettier/prettier/issues/5588